packages/lodestar/test/e2e/node.test.ts

```diff
@@ -24,7 +24,7 @@ describe("beacon node", function () {
 
   it("should finalize block", async function () {
     this.timeout(120000);
-    const bn = await getDevBeaconNode({SECONDS_PER_SLOT: 2, SLOTS_PER_EPOCH: 8});
+    const bn = await getDevBeaconNode({SECONDS_PER_SLOT: 2, SLOTS_PER_EPOCH: 8}, {sync: {minPeers: 0}});
```

Comment:
Should work without minPeers: 0 since you don't need to gossip if you don't have peers :slightly_smiling_face: